libdrgn/python/module.c

```diff
@@ -285,6 +285,24 @@ static int Module_set_build_id(Module *self, PyObject *value, void *arg)
 	return 0;
 }
 
+static DrgnObject *Module_object(Module *self, void *arg)
+{
+	struct drgn_error *err;
+
+	struct drgn_program *prog = drgn_module_program(self->module);
+	Program *prog_obj = container_of(prog, Program, prog);
```

Comment:
There are enough places that do this that we probably need a `Program *Module_prog(Module *module)` akin to `DrgnObject_prog()` and `DrgnType_prog()`.